src/angularComponent/ngcOmniboxController.js

```diff
@@ -635,6 +638,16 @@ export default class NgcOmniboxController {
           const choice = this.highlightedChoice;
           this.highlightNextChoice();
           this.unchoose(choice, false);
+        } else if (!isModifierKey(event)) {
+          // We can assume the user's keypress isn't meant to modify the highlighted choice and
+          // they instead just want to type as normal on the field, so we'll deselect the choice
+          // and focus the field which allows the user to type as normal
+          this.highlightedChoice = null;
+          this._fieldElement.focus();
+          // The field didn't have focus on keydown so it wasn't able to write out the character
+          // the user pressed, so we need to do that manually
```

Comment:
@onlywei look at line 608. The logic compares was the character position was at keydown vs keyup. Only after that if statement fails do I want to jump over to the field. I can't do this logic on keydown.